src/main/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImpl.java

```diff
@@ -113,6 +113,18 @@ public CompanyInsolvency retrieveCompanyInsolvency(String companyNumber) {
         return insolvencyDocument.getCompanyInsolvency();
     }
 
+    @Override
+    public void deleteInsolvency(String contextId, String companyNumber) {
+        logger.info(String.format(
```

Comment:
Maybe moving this log message to be after the deleteById repository call

Reply:
Not in scope for this ticket but in DSND-623 - the call to delete might be different as well. The only purpose of this log is to ensure QAs can see that the endpoint is being called